hoomd/md/pytest/test_flags.py

```diff
@@ -0,0 +1,81 @@
+import hoomd
+import pytest
+import numpy
+import itertools
+
+
+@pytest.fixture(scope='session')
+def md_simulation_factory(device):
+    def make_simulation(particle_types=['A']):
+        s = hoomd.Snapshot(device.comm)
+
+        # initialize particles on a simple cubic lattice and perturb them
+        # slightly off the lattice positions
+        # TODO: When hpmc_integrator_api is merged, port this functionality
```

Comment:
Do we want to go ahead and port this now since this is on that branch?

Reply:
Done.